main.nf

```diff
@@ -351,6 +356,75 @@ process reduce_cdnas {
     """ 
 }
 
+// build a assets for splici trancritome
+
+process build_splici_txome {
+ 
+    errorStrategy { sleep(Math.pow(2, task.attempt) * 200 as long); return  task.exitStatus == 130 || task.exitStatus == 137 || task.attempt < 3  ? 'retry': 'ignore' }
+    maxRetries 3
+    
+    conda "${baseDir}/envs/refgenie.yml"
+
+    input:
+        val(reduced) from REDUCED_REFERENCES
+        tuple val(species), val(assembly), file(filePath), val(additionalTag_genome), val(version), file(gtf), val(additionalTags) from SPLICI_BUILD_INPUTS.combine( GTF_SPLICI_BUILD_INPUTS.map{r -> tuple(r[0], r[1], r[2], r[5], r[6])}, by: [0, 1])
+       
+    output:
+        tuple val(species), val(assembly), val(version), val(additionalTags) into SPLICI_REFERENCE
+      
+    """
+    gunzip -f $filePath 
+    gunzip -f $gtf
+    pyroe make-splici *.fa *.gtf 90 splici --filename-prefix ${species}.${assembly}.
+    gzip splici/*.fa
+    
+    build_asset.sh \
```

Comment:
please confirm this and other scripts in the `/bin` folder do not need any modification to allow salmon splici indexes in Refgenie.